src/routes/SignIn.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { useFormik } from "formik";
+import {
+  Button,
+  Container,
+  TextField,
+  Typography,
+  Link,
+  Card,
+  Grid,
+} from "@material-ui/core";
+import { formStyles } from "../FormStyles";
+import { signInValidationSchema } from "../FormValidation";
+
+const SignIn = () => {
```

Comment:
```suggestion
export const SignIn = () => {
```

Reply:
So, I got this error "Attempted import error: './routes/SignIn' does not contain a default export (imported as 'SignIn')." when I initially made this change and then realized that I had to go into App.js and add curly braces. (applied this update to 'Register' as well.) :)